Yoast/Yoast-SEO-for-TYPO3
# [FEATURE] Added StructuredData API

## Summary

This PR can be summarized in the following changelog entry:

* Added new StructuredData API that will render the schema.org definitions for Site and BreadcrumbList.

## Relevant technical choices:

* Based on the PageTitle API of TYPO3 core

## Test instructions

This PR can be tested by following these steps:

* Make sure to remove all other possible schema.org generators in your installation
* Just install and check source of several pages
* You can check the syntax of the output on https://search.google.com/structured-data/testing-tool/u/0/


## Quality assurance


Fixes #149
